resources/views/message.blade.php

```diff
@@ -1,10 +1,10 @@
 <div class="m-2">
-    <div class="w-full {{ $headerStyle }} max-w-100"></div>
-    <div class="w-full {{ $headerStyle }} px-2 max-w-100">
+    <div class="w-full {{ $headerStyle }}"></div>
```

Comment:
We need that `max-w-100` to display regular error messages correctly on wide screens. Could you revert this change?